core/src/com/unciv/logic/battle/Battle.kt

```diff
@@ -258,7 +258,10 @@ object Battle {
             return
 
         var XPModifier = 1f
+        // keep until deprecated
         if (thisCombatant.getCivInfo().hasUnique("Military units gain 50% more Experience from combat")) XPModifier += 0.5f
+        for (unique in thisCombatant.getCivInfo().getMatchingUniques("Military units gain []% more Experience from combat"))
```

Comment:
If you're already parametrizing this, we can turn the Mitilary units into [] units, to allow specific unit types to gain more exp for specific civs or from buildings

Reply:
Understood. Should I close this pull request and open a new one?